src/extension/app/components/plugin/plugin.js

```diff
@@ -257,13 +266,29 @@ export class Plugin {
         `;
       }
 
+      if (this.isPopover() && this.config.url) {
+        const src = new URL(this.config.url);
+        src.searchParams.set('theme', this.appStore.theme);
+        return html`
+          <overlay-trigger receivesFocus="true" offset="-3">
+            <sk-action-button quiet slot="trigger">${this.getButtonText()}</sk-action-button>
+            <sp-popover slot="click-content" placement="top" tip style=${ifDefined(this.config.popoverRect)}>
```

Comment:
I think we should only keep `width` and `height` properties of the `DOMRect` object.